src/cli/onefuzz/status/top_view.py

```diff
@@ -78,30 +89,32 @@ def __init__(self, screen: Any, cache: TopCache, show_details: bool):
             "tasks": {"height": Widget.FILL_FRAME, "setup": self.cache.TASK_FIELDS},
             "messages": {
                 "height": min(10, max_widget_height),
-                "setup": [["Updated", "Type", "Message"], [1, 1, "100%"]],
+                "setup": ["Updated", "Type", "Message"],
             },
-            "status": {"height": 1, "setup": [None, [1, "100%"]]},
+            "status": {"height": 1},
         }
 
         for name in ["status", "pools", "jobs", "tasks", "messages"]:
             if name == "messages" and not self.show_details:
                 continue
 
-            if dimensions[name]["setup"][0]:
+            titles = dimensions[name].get("setup")
```

Comment:
I'm assuming this is typechecked, so the use sites below work as expected.

But even so, when reading, the possible values can be hard to keep straight because we are doing truthy checks on what I'm guessing is an `Optional[List[T]]`, for some `T`.

All that said, should we maybe just default `titles` to an empty list?